Makefile

```diff
@@ -118,3 +119,20 @@ dist: clean docs assets
 
 install: clean
 	python setup.py install
+
+pex:
+	ls dist/ka_lite-*.whl | while read whlfile; do pex $$whlfile -o dist/kalite-`unzip -p $$whlfile kalite/VERSION`.pex -m kalite --python-shebang=/usr/bin/python; done
+
+
+dockerwriteversion:
+	python -c "import kalite; print(kalite.__version__)" > kalite/VERSION
```

Comment:
Please remove this step, for the reason stated above (reading it back out via `unzip` is heavy) + introducing some new version maintaining mechanism isn't really ideal.

You can use `git describe --tags` or  `python -c "import kalite; print(kalite.__version__)"`